src/schema/v2/artwork/collectorSignals.ts

```diff
@@ -3,8 +3,6 @@ import { GraphQLInt, GraphQLObjectType } from "graphql"
 import { ResolverContext } from "types/graphql"
 import { PartnerOfferToCollectorType } from "../partnerOfferToCollector"
 import { isFeatureFlagEnabled } from "lib/featureFlags"
-import { some } from "lodash"
-import { isFieldRequested } from "lib/isFieldRequested"
```

Comment:
Yah, i was thinking in the other PR that if there's anything at all uncommon being used, we should second guess! Thats not to say we're not to think out of the box but rather to emphasize the already-powerful patterns we have at our disposal.